src/EuroDNS/Helper/EuroDNSApi.php

```diff
@@ -1029,6 +1043,7 @@ public function setManualNS($type, $domainName)
         $i = 1;
         // Loop through name servers and generate XML
         foreach ($nameserversArray as $key => $val) {
+            // @phpstan-ignore-next-line
```

Comment:
@uphlewis I think `$val` is always a string, unless `target` can return values other than `string`